spk/rutorrent/src/service-setup.sh

```diff
@@ -181,12 +190,29 @@ service_restore ()
 {
     # Restore the configuration file
     mv -f "${TMP_DIR}/config.php" "${WEB_DIR}/${PACKAGE}/conf/" >>"${INST_LOG}" 2>&1
+    set_unix_permissions "${WEB_DIR}/${PACKAGE}/conf/config.php"
```

Comment:
Shouldn't this be done after the configuration file is updated?

Reply:
Well this is overwriting the default provided configuration file.  I believe (but may be wrong) the `service_postint` is not being invoked when "updating" a package, thus the configuration file is default until being overwritten by the recovery from the previous version.

If you where specifically referring to the `set_unix_permissions`, modifications done afterwards by appending the `define_external_program` for missing parameters when porting over an old configuration but that doesn't affect its permissions.  Similar to:
```
$ sudo touch toto
$ sudo chown sc-rutorrent:rutorrent toto
$ echo "AAA" | sudo tee -a toto
$ ll toto
-rw-r--r-- 1 sc-rutorrent rutorrent 4 Dec 10 10:46 toto
```

Reply:
Actually the service_postinst is supposed to be invoked but some of its steps are only executed when not upgrading. Indeed one of the impacts is that the default configuration file gets overwritten by the one "in place". 
I was actually referring to the authorizations logics but indeed if ordering has no impact then my point was "moot" ;)

Reply:
You are totally right about `service_postint`... and re-reading the code makes it now obvious, thnx for the explanations :)